### DISCUSSION TITLE
Ability to drag value(s) from Results pane to Editor pane

### DISCUSSION THREAD

**Discussion by gregclerk** [microsoft/azuredatastudio#23670] [Ideas]
Is SSMS you can highlight one or more cells in the results grid and drag/drop them onto the editor section above without having to copy to your clipboard.  Can this be added to Azure Data Studio?

**Comment by erinstellato-ms** [comment]
Hi @gregclerk, as this is a feature request, could you please open an issue so that it can be triaged and added to our backlog?  This keeps all requests in one location for other users to search and upvote.  Thanks!

**Reply by gregclerk** [reply]
You mean the "Ideas" category, right?

**Reply by erinstellato-ms** [reply]
I mean go to the Issues tab in the repo, create a new issue, then select the Get started button next to feature request.

![image](https://github.com/microsoft/azuredatastudio/assets/98599126/44801b60-d4d6-4487-9247-2be1e89c353b)
